compiler/rustc_lint/messages.ftl

```diff
@@ -196,7 +196,7 @@ lint_drop_trait_constraints =
 
 lint_dropping_copy_types = calls to `std::mem::drop` with a value that implements `Copy` does nothing
     .label = argument has type `{$arg_ty}`
-    .note = use `let _ = ...` to ignore the expression or result
+    .note = use `{$replace}` to ignore the value instead of dropping directly
```

Comment:
Why not just make this a `#[suggestion]`?

Reply:
It would need to be `maybe-incorrect`, because we don't want users to blindly apply the suggestion without first checking their code for a possible logic error.

Reply:
Of course, but `maybe-incorrect` exists exactly for this reason. I'd rather we not manually implement a "suggestion-like" note when we have the infrastructure to render suggestions.

Reply:
Done. Thank you.